sprint-65-summary.md (#2380)
raft-tech/TANF-app: docs/Sprint-Review/sprint-65-summary.md

# Sprint 65 Summary
1/18/23-1/31/23

Velocity: 21

> ## Sprint Goal

Reviewer: ```suggestion
## George Hudson, devops engineer, joined the TDP team on 1/23/23.  
Sprint Goal
```

* Work on TANF Section 1 data parsing, complete submission history MVP functionality while addressing the system memory limitations for the first V3 release. Additionally, complete preparations for TDP user expansion in time for the FY23 Q1 data submission.
 

## Tickets
### Closed/Merged/Completed (Done/Demo, Closed)
* [#2335 SPIKE: Evaluate parser validation methods](https://github.com/raft-tech/TANF-app/issues/2335)
* [#2141 As technical lead, I want an authenticated-user integration test](https://github.com/raft-tech/TANF-app/issues/2141)
* [#217 As a system admin, I want the STT object models to be revised](https://github.com/raft-tech/TANF-app/issues/217)

> * [#1354 Implement storage of parsed data in Elasticsearch](https://github.com/raft-tech/TANF-app/issues/1354)

Reviewer: This was closed because it was covered elsewhere. Suggest removing from this list since the work isn't complete. Maybe we should remove "closed" from header" because it should just be what is completed. Thoughts?

Author: @lfrohlich Hey Lauren, agree re: complete vs closed - will update the template to delineate there!

# Sprint 65 Summary
1/18/23-1/31/23

Velocity: 21

> ## George Hudson, devops engineer, joined the TDP team on 1/23/23.  

Reviewer: ```suggestion
George Hudson, devops engineer, joined the TDP team on 1/23/23.  
```

> Sprint Goal

Reviewer: ```suggestion
## Sprint Goal
```

 

## Tickets
### Closed/Merged
* [#2335 SPIKE: Evaluate parser validation methods](https://github.com/raft-tech/TANF-app/issues/2335)
* [#2141 As technical lead, I want an authenticated-user integration test](https://github.com/raft-tech/TANF-app/issues/2141)
* [#217 As a system admin, I want the STT object models to be revised](https://github.com/raft-tech/TANF-app/issues/217)
* [#2298 [Design Deliverable] Update Project Updates site with Oct-2.2 content](https://github.com/raft-tech/TANF-app/issues/2298)
* [#2360 [WIP] test staging and production deployment before v3.0 release](https://github.com/raft-tech/TANF-app/issues/2360)

### Ready to Merge
* N/A

### Submitted (QASP Review, OCIO Review)

* [#2350 Increase memory limit for deployed instances and celery workers](https://github.com/raft-tech/TANF-app/issues/2350)
* [#132 TANF Active Data (01) File Parsing - As a data analyst, I want my data files to be parsed so that my submission can be validated/accepted by TDP.](https://github.com/raft-tech/TANF-app/issues/132)

## Closed (not merged)
* [#1354 Implement storage of parsed data in Elasticsearch](https://github.com/raft-tech/TANF-app/issues/1354)

> * Covered in scope of #1351 and #132, aligned to close in BR across OFA/Raft

Reviewer: ```suggestion
* Covered in scope of #1351 and #132, agreed to close in backlog refinement across OFA/Raft
```
